src/app/Processor/ProcessorEdit/ConfigurationForm/ConfigurationForm.tsx

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-assignment,@typescript-eslint/no-unsafe-return  */
```

Comment:
Silencing a bunch of linting rules here because of some `any` coming from the deps. Not much we can do about it.